Chapters/1-4-Chapter4.md

```diff
@@ -0,0 +1,232 @@
+
+"Laen!" I yelled at him. If I'm being honest, I wanted to get mine first.
+
+I caught up to him as he was begging Gilli for a cake.
+
+"I'm sorry, Gilli. I promised him I would get him one."
+
+She laughed and said, "Oh it's alright. Everyone in Heiko deserves a cake, even saffas. Here!"
+
+She tossed a roasted tor-nut handcake into the air and Laen caught it with his mouth with impeccable precision.
+
+"Could I bother you for another for myself?"
+
+"Just keep those pohua shipments coming in," she said, and handed me one of the same kind.
+
+I reached out to take it, but she didn't let go.
+
+"I mean it," she said under her breath, "I need more. That last shipment you gave me was fantastic. It makes a better syrup than idze."
+
+I looked at her and smiled uncomfortably, slightly intimidated, "Y-yea... I'll... See what I can do."
+
+Her smile returned, and she said, "Thank you, Liffa... Next!"
+
+I took one bite and tasted the juicy sweetness. She wasn't wrong. They had gotten better than I remembered them being. But then a dark shadow came over my hand and swooped down to steal my cake.
+
+"Laen! That was mine!" I yelled, and gave him a slap on the shoulder. He barely noticed through his chewing.
+
+"Once you're done, go back home ok?"
+
+He didn't respond, but I knew he'd heard me. I left him in at the corner, humming with his stolen cake and a handful of water from my canteen, and made my way to the platform.
+
+My feet found some rest squatting and leaning against a tree near the tracks, hiding from the wind and from Kipnu if he had decided to follow me. _This is the right thing_, I affirmed to myself. _No externalities. No liabilities_. I thought about the crops I had just planted. Whether Melna would ask Tiggu to work on them. Whether she would kill them out of spite, or replant them in her own field. If this is as important as Jathi said, then those crops don't matter in the scheme of things. Heiko would understand.
+
+The station attendant called for us through a flathorn, "Train to Shaki approaching. All passengers prepare to marshal."
+
+I quickly picked up my bag and jumped the wall to the marshalling side. The attendant was tall for a maman with a recent flow tattoo mutating into hundreds of threads of infrared across hae neck. I admired it from afar as I shifted into the line for the 9th car. I had gotten a flow tattoo once, but working in the constant light of Ara made my skin's water vessels flow too quickly, so it all mixed together and just looked like I had some weird genetic defect in my hydroderm. My parents just laughed and told me about how they had tried it too when they were kids, and had the same results.
```

Comment:
So flow tattoos are when they release ink into the water of the skin?

Reply:
Yep! I was trying to think of a better word for this, but nothing came to mind.